### DISCUSSION TITLE
"Crop input image based on A1111 mask" not working?

### DISCUSSION THREAD

**Discussion by lverangel** [Mikubill/sd-webui-controlnet#2513] [Q&A]
sorry for english
In my mind, this setting can automatically cut the hand picture of controlnet,
But I don't see that happening here,
how can i make it, 
or how can i find the log and what's the params sent to controlnet?
![1](https://github.com/Mikubill/sd-webui-controlnet/assets/2209518/4fef3b66-0026-4dbb-90a1-a64b80d604a2)
![2](https://github.com/Mikubill/sd-webui-controlnet/assets/2209518/7a4f38bc-f0c0-465f-9641-db9a539c71e1)
![3](https://github.com/Mikubill/sd-webui-controlnet/assets/2209518/ca4c569e-bf73-4673-935e-13c23a3e217b)

**Comment by huchenlei** [comment]
You can add `--controlnet-loglevel DEBUG` commandline args to the launch script.

You should see
```
2024-01-20 10:27:05,565 - ControlNet - DEBUG - A1111 inpaint mask START
2024-01-20 10:27:05,643 - ControlNet - DEBUG - A1111 inpaint mask END
```
during generation when `Crop input image based on A1111 mask` is selected.

It is also often easier to reason if you can align the dimension of control image and the image you want to inpaint.

**Reply by lverangel** [reply]
![4](https://github.com/Mikubill/sd-webui-controlnet/assets/2209518/5d693e34-748b-43f4-922b-360e5491aee4)
there is no log like this, any ideas?

version: [v1.7.0] •  python: 3.10.8  •  torch: 2.1.0+cu121  •  xformers: 0.0.22.post7  •  gradio: 3.41.2

**Reply by huchenlei** [reply]
This seems wield, it seems like the checkbox is selected but the value is not being used. I cannot reproduce this locally though.
